docs/articles/documentation/reference/api/browserconnection/url.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: docs
+title: BrowserConnection.url Property
+permalink: /documentation/reference/api/browserconnection/url.html
+---
+# BrowserConnection.url Property
+
+Returns A URL that should be visited from a remote browser in order to connect it to the TestCafe server.
```

Comment:
Returns a URL to be visited from a remote browser to establish a BrowserConnection.

?